Editor/Timeline/CinemachineShotEditor.cs

```diff
@@ -51,9 +51,10 @@ static SyncCacheEnabledSetting()
         }
 #endif
 
-        static public CinemachineVirtualCameraBase CreateStaticVcamFromSceneView()
-        {
-            CinemachineVirtualCameraBase vcam = CinemachineMenu.CreateStaticVirtualCamera();
+        static public CinemachineVirtualCameraBase CreatePassiveVcamFromSceneView()
+       {
+            CinemachineEditorAnalytics.SendCreateEvent("Virtual Camera");
```

Comment:
This is new analytics. The Create Event analytics is specifically for the Create Menu. I don't think this is wanted.